src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/hooks/useFeedbackReviews.ts

```diff
@@ -0,0 +1,152 @@
+
+  const { fetchReviewsData } = useFeedbackReviewsData();
+
+  const getReviewsAsync = useAsync(getReviewsApi, async (result) => {
+    const reviewsData = await fetchReviewsData({
+      reviews: result?.data?.result ?? [],
+      userId: user?.id ?? '',
+    });
+    setReviewerData(reviewsData);
+  });
+
+  const getFlowReviewsAsync = useAsync(getFlowReviewsApi, async (result) => {
+    const reviewsData = await fetchReviewsData({
+      reviews: result?.data?.result ?? [],
+      userId: user?.id ?? '',
+    });
+    setReviewerData(reviewsData);
+  });
+
+  const { execute: getReviews, isLoading: reviewsLoading, error: reviewError } = getReviewsAsync;
+  const {
+    execute: getFlowReviews,
+    isLoading: reviewsFlowLoading,
+    error: reviewsFlowError,
+  } = getFlowReviewsAsync;
+
+  const updateAssessment = (updatedAssessment: AssessmentActivityItem[]) => {
+    setAssessment(updatedAssessment);
+    reset(getDefaultFormValues(updatedAssessment));
+    setAssessmentStep(0);
+  };
+
+  const handleSelectLastVersion = () => {
+    setIsLastVersion(true);
+    setIsBannerVisible(false);
+
+    const updatedAssessment = lastAssessment?.map((activityItem) => ({
+      activityItem,
+      answer: undefined,
+    })) as AssessmentActivityItem[];
+    updateAssessment(updatedAssessment);
+  };
+
+  const handleGetReviews = useCallback(async () => {
+    if (!appletId) return;
+
+    if (answerId) {
+      await getReviews({ appletId, answerId });
+    } else if (submitId) {
+      await getFlowReviews({ appletId, submitId });
+    }
+  }, [appletId, answerId, submitId, getReviews, getFlowReviews]);
+
+  const handleReviewerAnswersRemove = async (assessmentId: string) => {
+    if (!appletId) return;
+
+    if (answerId) {
+      try {
+        setRemoveReviewsLoading(true);
+        setRemoveReviewsError(null);
+
+        await deleteReviewApi({ appletId, answerId, assessmentId });
+      } catch (error) {
+        setRemoveReviewsError(getErrorMessage(error));
+      } finally {
+        setRemoveReviewsLoading(false);
+      }
+    } else if (submitId) {
+      try {
+        setRemoveReviewsFlowLoading(true);
+        setRemoveReviewsFlowError(null);
+
+        await deleteFlowReviewApi({ appletId, submitId, assessmentId });
+      } catch (error) {
+        setRemoveReviewsFlowError(getErrorMessage(error));
+      } finally {
+        setRemoveReviewsFlowLoading(false);
+      }
+    }
+
+    if (lastAssessment?.length) {
+      handleSelectLastVersion();
+    } else {
+      const updatedAssessment = assessment?.map(({ activityItem, items }) => ({
+        activityItem,
+        answer: undefined,
+        items,
+      })) as AssessmentActivityItem[];
+      updateAssessment(updatedAssessment);
+    }
+    handleGetReviews();
+  };
+
+  return {
+    isBannerVisible,
+    setIsBannerVisible,
+    reviewerData,
+    reviewsLoading,
+    reviewsFlowLoading,
+    reviewsError: reviewError ? getErrorMessage(reviewError) : null,
```

Comment:
Could you please return a single `reviewsError` that includes both `reviewError` and `reviewsFlowError`? Please, do the same for `removeReviewsLoading` and `removeReviewsError`.